index.d.ts

```diff
@@ -87,7 +85,7 @@ declare class Long {
     static fromBytes(bytes: number[], unsigned?: boolean, le?: boolean): Long;
 
     /**
-     * Creates a Long from its little endian byte representation.
+     * Creates a Long from its big endian byte representation.
```

Comment:
Why has this been switched? fromBytesLE = little endian, fromBytesbE = big endian

Reply:
Sorry wanted to update documentation for the other function. Have fixed it.